### PR TITLE
Code for filtering missing values

### PR DESCRIPTION
**Pull request by ahmostafa147**
An SQL file for the code that gets the patients that have their weight recorded.

### CODE REVIEW

File: sql_queries/utilities/filter_patients_no_weight.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    MAX(
+        CASE 
+            WHEN c.itemid IN (762, 763, 3723, 3580, 3581, 3582, 226512) 
+                AND c.valuenum IS NOT NULL 
+            THEN 1 
+            ELSE 0 
+        END
+    ) AS any_weight_recorded
+FROM 
+    unique_p p
```
Comment (Sherrihuu) [inline sql_queries/utilities/filter_patients_no_weight.sql]: What is unique_p? How to access it? Please change it before merging

File: sql_queries/utilities/filter_patients_no_weight.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+LEFT JOIN 
+    chartevents c 
+    ON p.subject_id = c.subject_id 
+    AND p.hadm_id = c.hadm_id 
+    AND p.icustay_id = c.icustay_id
+WHERE 
+    c.itemid IN (762, 763, 3723, 3580, 3581, 3582, 226512)
```
Comment (Sherrihuu) [inline sql_queries/utilities/filter_patients_no_weight.sql:48]: What are these values? maybe add a comment explaining this.

File: sql_queries/utilities/filter_patients_no_weight.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
+SELECT 
+    p.subject_id, 
+    p.hadm_id, 
+    p.gender, 
+    MAX(p.age_at_admission) AS max_age_at_admission,
```
Comment (Sherrihuu) [inline sql_queries/utilities/filter_patients_no_weight.sql]: Why are you using max?

**Review by Sherrihuu — CHANGES_REQUESTED** [review]
I think the code looks good but needs some extra comments to follow the convention we agreed on!

**Review by Bobby-Zhu — COMMENTED** [review]
I looked over sherry's comments and I don't spot any of other problems

File: sql_queries/utilities/filter_patients_no_weight.sql (hunk 1 above)
Comment (ahmostafa147) [reply sql_queries/utilities/filter_patients_no_weight.sql]: added the code for it and added a comment.

File: sql_queries/utilities/filter_patients_no_weight.sql (hunk 2 above)
Comment (ahmostafa147) [reply sql_queries/utilities/filter_patients_no_weight.sql:48]: Added description for the ids.

File: sql_queries/utilities/filter_patients_no_weight.sql (hunk 3 above)
Comment (ahmostafa147) [reply sql_queries/utilities/filter_patients_no_weight.sql]: Added description

**Pull request merged**